libgloss/arc/hl/hl_getcwd.c

```diff
@@ -0,0 +1,62 @@
+/*********************************************************************
+Copyright  2002-2020 Synopsys Inc.
```

Comment:
header, comments, codestyle

Reply:
dropped it -- newlib does not provide getcwd.